### PR TITLE
LPS-182795 Prepare sidebar to edition of new field type 

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
Tasks :
[LPS-182795](https://issues.liferay.com/browse/LPS-182795)
[LPS-182796](https://issues.liferay.com/browse/LPS-182796)

This PR needs this other to be merge :
https://github.com/liferay-objects/liferay-portal/pull/2028

### CODE REVIEW

**Comment by naraandrade** [comment]
Just started reviewing :)

**Review by naraandrade — APPROVED** [review]

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/MaxLengthProperties.tsx [hunk 1]
```diff
@@ -44,7 +44,10 @@ export function MaxLengthProperties({
 	setValues,
 }: IMaxLengthPropertiesProps) {
 	const [defaultMaxLength, defaultMaxLengthText] =
-		objectField.businessType === 'Text' ? [280, '280'] : [65000, '65,000'];
+		objectField.businessType === 'Text' ||
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/MaxLengthProperties.tsx:47]: @pedro-oliveira446 Just a small change, keep it in alphabetical order like:
`objectField.businessType === 'Encrypted' ||
objectField.businessType === 'Text' ? [280, '280']
			: [65000, '65,000'];`

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Review by naraandrade — APPROVED** [review]

**Comment by p-albuquerque** [comment]
ci:forward:force

**Pull request closed without merging**